Build fails running Build.cmd at RGBLedMatrix/samples/Program.cs - CS4014

**Describe the bug**
Build fails running Build.cmd at RGBLedMatrix/samples/Program.cs(66,1)
`error CS4014: Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.
`
**Steps to reproduce**
Running Build.cmd on Windows command line.

Dotnet --version: 7.0.401
(instead of 7.0.203 as configured in the most recent global.json)

--- a/global.json
+++ b/global.json
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "7.0.203",
+    "dotnet": "7.0.403",
     "runtimes": {
       "dotnet/x64": [
         "6.0.16"
@@ -11,7 +11,9 @@
     }
   },
   "sdk": {
-    "version": "7.0.203"
+    "version": "7.0.403",
+    "allowPrerelease": true,
+    "rollForward": "major"
   },
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.23517.4",

--- a/src/devices/RGBLedMatrix/samples/Program.cs
+++ b/src/devices/RGBLedMatrix/samples/Program.cs
@@ -63,7 +63,7 @@ RGBLedMatrix matrix = new RGBLedMatrix(mapping, 32, 32);
 
 // If you chain 4 32x32 panels having 2 rows chaining (2 panels in first row an d2 panels in second row).
 // RGBLedMatrix matrix = new RGBLedMatrix(mapping, 64, 64, 2, 2);
-Task.Run(() =>
+_ = Task.Run(() =>
 {
     matrix.StartRendering();
 

